build-tools/packages/build-cli/docs/generate.md

```diff
@@ -142,6 +143,29 @@ EXAMPLES
     $ flub generate changeset --all
 ```
 
+## `flub generate manifest`
+
+As all builds are published to build feed, this command picks the latest successful build against the provided branch name and generates dev manifest file.
+
+```
+USAGE
+  $ flub generate manifest -o <value> -p <value> -t <value> -s <value> [-v | --quiet]
+
+FLAGS
+  -o, --organization=<value>  (required) Organization name
+  -p, --project=<value>       (required) Project name
+  -s, --sourceBranch=<value>  (required) Branch name across which the dev release manifest should be generated.
+  -t, --ado_pat=<value>       (required) ADO Personal Access Token
```

Comment:
Remove all the one-character aliases. No need to add them since this command will usually run in CI.